### DISCUSSION TITLE
startup/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#44] [Announcements]
# startup/

It Happened Again.

https://www.supergoodcode.com/startup/

**Comment by qwertychouskie** [comment]
Now what I wanna know, is whether you can make Iris faster as well :eyes:

**Comment by Archprogrammer** [comment]
Boiling the pasta for a few minutes will make it go down *faster...*

**Comment by Permik** [comment]
# So uhhh, Mike...
![picture of the Y\'all Got Any More Of That -meme, of a man scratching their neck, looking like they\'re about to ask you for more of something you've provided them earlier, their lips are white from a supposed powder that you provided them earlier, they look like they\'re eager to get some more of that same good stuff ](https://i.kym-cdn.com/entries/icons/original/000/015/231/Y'all_got_anymore_meme_banner.jpg)
 _Y'all got any more of them blog posts?_

**Reply by qwertychouskie** [reply]
I am deprived of spaghett @zmike

**Reply by zmike** [reply]
LET ME COOK

**Reply by qwertychouskie** [reply]
@zmike *nudge* https://gitlab.freedesktop.org/mesa/mesa/-/issues/7287#note_1558063 *nudge*

**Reply by zmike** [reply]
I think Venemo was pretty clear in asking for games that exhibited the issue.
